### PR TITLE
Fix case application process exit immediately after launching process(.exe file)

### PR DESCRIPTION
**Pull request by quangnt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,10 @@ Contributions are welcome!
 3. We recommend to write a test which shows that the bug was fixed or that the feature works as expected.
 4. Send a pull request and bug the maintainer until it gets merged and published. :smiley:
 
+## Fix issue there're some applications that launched process exists immediately(Ex: microsoft calculator, firefox, chrome, iexplorer), so in close/quit function it throws exception Process Not Found.
```
Comment (quangnt) [inline README.md]: from @zelonght 
(All these changes) This should be somewhere like changelog, not in README.md

File: src/Winium.Cruciatus/Application.cs [hunk 2]
```diff
@@ -109,9 +122,37 @@ public void Start(string arguments)
                                Arguments = arguments
                            };
 
-            this.process = Process.Start(info);
+            this.process = Process.Start(info);
```
Comment (quangnt) [inline src/Winium.Cruciatus/Application.cs]: from @zelonght : space issue here

File: src/Winium.Cruciatus/Application.cs [hunk 3]
```diff
@@ -109,9 +122,37 @@ public void Start(string arguments)
                                Arguments = arguments
                            };
 
-            this.process = Process.Start(info);
+            this.process = Process.Start(info);           
         }
 
+        /// <summary>
+        /// Update process property by process name
+        /// </summary>
```
Comment (quangnt) [inline src/Winium.Cruciatus/Application.cs]: from @zelonght This is like hardcoded, any in future we may have other similar apps like this to append?
		Any general solution for this? Should do simple way, int case many process return without doing anything

**Comment by quangnt** [comment]
from @zelonght: 
No description provided?
PR title: "Add HasExited() and UpdateRunApplicationProcessBy() methods" -- why add those methods? I think PR's title should tell what the PR for. Same for commit text.
I had some inline comments.
There is a test case failed @ http://opensource-ci.2gis.ru/job/Winium.Cruciatus-PRs/70/ please address

File: src/Winium.Cruciatus/Application.cs [hunk 4]
```diff
@@ -109,9 +122,37 @@ public void Start(string arguments)
...
+            {
+                return;
+            }
+
+            var processList = Process.GetProcessesByName(launchedAppProcessName);
+            if (processList.Length >= 1)
+            {
+                // Get running process base on StartTime          
+                var processSortedList = processList.OrderBy(p => p.StartTime);
+                if ((launchedAppProcessName.ToLower().Equals("iexplore")) || (launchedAppProcessName.ToLower().Equals("firefox")) || (launchedAppProcessName.ToLower().Equals("chrome")))
```
Comment (quangnt) [inline src/Winium.Cruciatus/Application.cs]: from @zelonght This is like hardcoded, any in future we may have other similar apps like this to append?
Any general solution for this? Should do simple way, int case many process return without doing anything

File: src/Winium.Cruciatus/Application.cs (hunk 2 above)
Comment (zelonght) [inline src/Winium.Cruciatus/Application.cs]: @quangnt You have not addressed this

File: src/Winium.Cruciatus/Application.cs [hunk 5]
```diff
@@ -109,9 +120,32 @@ public void Start(string arguments)
...
-            this.process = Process.Start(info);
+            this.process = Process.Start(info);           
         }
 
+        /// <summary>
+        /// Update process property by process name
+        /// </summary>
+        /// <param name="processName">Launched application process name</param>
+        /// <returns>true if launched process is successfully update. false if there is error occurs</returns>
+        public void UpdateProcessBy(string processName)
```
Comment (zelonght) [inline src/Winium.Cruciatus/Application.cs]: To match the function description `Update process property by process name` then let's name the function as its meaning `UpdateProcessBy` --> `UpdateProcessByName`

**Comment by quangnt** [comment]
@zelonght I already addressed white space redundancy in Application.cs(line 125) and rename function to UpdateProcessByName() as you suggested. Please review again. Thanks

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,7 +1,11 @@
 ﻿# Change Log
 
 <!--## Unreleased-->
+## v2.10.1
 
+- Fix issue there're some applications that launched process exists immediately(Ex: microsoft calculator, firefox, chrome, iexplorer), so in close/quit function it throws exception Process Not Found.
```
Comment (zelonght) [inline CHANGELOG.md]: @quangnt Please make this brief just like the author is doing like https://github.com/PersonifyInc/Winium.Cruciatus/pull/1/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR12 and here https://github.com/PersonifyInc/Winium.Cruciatus/pull/1/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR17

File: src/TestApplications.Tests/WpfTestApplication.Tests/WpfTestApplication.Tests.csproj [hunk 7]
```diff
@@ -62,7 +62,9 @@
     <Compile Include="TestCases\CheckThirdTab.cs" />
     <Compile Include="TestClassHelper.cs" />
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
```
Comment (zelonght) [inline src/TestApplications.Tests/WpfTestApplication.Tests/WpfTestApplication.Tests.csproj]: Remove this please

File: src/TestApplications.Tests/WindowsFormsTestApplication.Tests/WindowsFormsTestApplication.Tests.csproj [hunk 8]
```diff
@@ -64,6 +64,9 @@
       <Name>Winium.Cruciatus</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
```
Comment (zelonght) [inline src/TestApplications.Tests/WindowsFormsTestApplication.Tests/WindowsFormsTestApplication.Tests.csproj]: Remove this please

**Comment by quangnt** [comment]
@zelonght Done. Thanks

File: src/TestApplications.Tests/WpfTestApplication.Tests/WpfTestApplication.Tests.csproj [hunk 9]
```diff
@@ -44,6 +44,7 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="CheckFindByXPath.cs" />
+    <Compile Include="CheckElementIsStaleProperty.cs" />
```
Comment (zelonght) [inline src/TestApplications.Tests/WpfTestApplication.Tests/WpfTestApplication.Tests.csproj]: Redundant?

**Comment by zelonght** [comment]
@quangnt Rebased, updated changelog, squashed into 1 commit to easier to rebase with upstream.

**Comment by zelonght** [comment]
:+1:

**Pull request merged**